examples/gno.land/r/gnoland/faucet/faucet.gno

```diff
@@ -43,7 +43,7 @@ func Transfer(to std.Address, send int64) string {
 	gTotalTransfers++
 
 	banker := std.GetBanker(std.BankerTypeRealmSend)
-	pkgaddr := std.GetOrigPkgAddr()
```

Comment:
why do we this change this? We want this request  from the end user in stead of from another realm.

Reply:
banker type used here is 'RealmSend', not 'OrigSend'
I think it should be CurrentRealm()